procedures/ExportWrite.java

```diff
@@ -11,8 +11,9 @@ public class ExportWrite extends VoltProcedure {
   // Arrays of the function, key, and value for each op in the transaction.
   // We assume string keys and integer values.
   public long run(int part, long[] elements) {
+    boolean flag = ((elements[0]%2)==0);
     for (int i = 0; i < elements.length; i++) {
-      if (Math.random() < 0.5) {
+     if (flag) {
```

Comment:
old code randomized every element.  new code uses same for every element.  not sure that was the intent

Reply:
It is not relevant.
1. Procedure can not have "random" generator. Otherwise operation against replicas fail.
2. what is important, the test tries both, stream first and table first.
The previous code had an obvious bug. It was concocted by Richard or Kyle.